doc/benchmark_workflow/run_benchmark.rst

```diff
@@ -0,0 +1,69 @@
+.. _run_benchmark:
+
+Run a benchmark
+===============
+
+Let's use the Lasso benchmark to illustrate ways of running a benchmark.
+Beforehand, make sure that benchopt is installed and the Lasso benchmark is set up by following the instructions in :ref:`get_started`.
+
+With the :ref:`cli_ref`, there are two options
+
+Directly in the terminal
+------------------------
+
+It is possible to specify the solvers as well as the datasets
+to include in the benchmark run by using flags after ``benchopt run .``.
+
+For instance, the following command runs the benchmark with solvers
+``skglm`` and ``celer``, on datasets "leukemia" and "simulated".
+
+.. prompt:: bash $
+
+    benchopt run . -s skglm -s celer -d leukemia -d simulated
+
+The ``-s`` flag is to specify a solver whereas ``-d`` specifies the dataset.
+To include multiple datasets/solvers, use multiple ``-d``/ ``-s`` flags.
+
+.. note::
+
+    The ``run`` command accepts other flags such ``-j`` to run the benchmark in parallel with a given number of processes.
+    The list of flags is available through ``benchopt run --help`` or in the :ref:`cli_ref` page .
+
+In addition, it is possible to specify the parameters of solvers and datasets by wrapping them in square brackets in comma separated format.
+
+Here is an example to run Proximal Gradient Descent (``Python-PGD``) with acceleration on simulated data with number of samples ``n_samples`` equal ``100`` and number of features ``n_features`` set to ``20``.
+
+.. prompt:: bash $
+
+    benchopt run . -s Python-PGD["use_acceleration"=True] -d simulated["n_samples=100","n_features"=20]
```

Comment:
```
(base) ➜  lasso git:(mnt) ✗ benchopt run . -s Python-PGD["use_acceleration"=True] -d simulated["n_samples=100","n_features"=20]
zsh: no matches found: Python-PGD[use_acceleration=True]
```
quotes must be outside. Also, let's write the syntax without default, and add a note for zsh users

Reply:
I have added a tab to exhibit both cases

![Screenshot from 2023-07-27 11-24-15](https://github.com/benchopt/benchopt/assets/65614794/d5ea9b6b-0385-46d5-95db-82cbef88cace)

WDYT, @mathurinm?

Reply:
looks good